src/backend/marsha/bbb/api.py

```diff
@@ -144,7 +147,7 @@ def get_serializer_context(self):
             )().has_permission(self.request, self)
         return serializer_context
 
-    def _get_list_queryset(self):
+    def _get_filtered_queryset(self):
         """Build the queryset used on the list action."""
```

Comment:
the docstring is not accurate anymore. `list` and `bulk_destroy` should be mentin here.